hyperSpec/vignettes/vignette-default-settings.R

```diff
@@ -68,12 +70,30 @@ set_trellis_layout_hw_custom <- function() {
 set_trellis_layout_hw_custom()
 
 # knitr Settings =============================================================
+
+# Notes on Figure Sizes.  There are (at least) 2 common figure aspect ratios in
+# the vignettes.  One is a full width figure, suitable for spectra or spectra-like
+# plots.  The other is a square plot, typically used by maps.
+# Figure size is controlled by the hook defined just below, and notice that
+# in the opts_chunk sq.fig = FALSE as the default.
+
+knitr::opts_hooks$set(sq.fig = function(options) {
+  if (isTRUE(options$sq.fig)) {
+    options$fig.width = 6
+    options$fig.height = 6
```

Comment:
@bryanhanson Did you base these defaults on `hyperSpec.Rmd` only?

In general, 6x6 plots are too large as they occupy almost the whole screen. There were some plots, which should be that big. But it's more exception than the rule.

Reply:
Yes, based on `hyperSpec.Rmd` only, that's the only one I have reviewed.

Reply:
Oh, one more thing.  The plots coming from lattice appear to have a great deal of white space around them.  Not sure if there is a `croppdf`-like hook that would apply to png plots.  Something else that could be investigated.

Reply:
While updating to the more flexible hook I set square figure dimensions to 5 x 5.